### PR TITLE
Get info before game

### PR DESCRIPTION
**Pull request by yummta**

### CODE REVIEW

File: frontend/src/views/flight-details.test.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// src/components/channel-list.test.js
+import React from "react";
+import { render } from "@testing-library/react";
+
+import FlightDetails from "./flight-details";
+
+test("FlightDetails component", () => {
+  const { asFragment } = render(<FlightDetails />);
+  expect(asFragment()).toMatchSnapshot();
+});
```
Comment (valevassallo) [inline frontend/src/views/flight-details.test.js:10]: I suggest testing the button you have in your component as well. As for now, that button does nothing, but as soon as you give it a functionality, then test it.

File: frontend/src/App.js [hunk 2]
```diff
@@ -1,7 +1,47 @@
...
+          body {
+            font-family: "Roboto", sans-serif;
+            line-height: 1.5;
+          }
+        `}
+      />
+      <div css={cssWrapper}>
+        <div
+          css={{
+            maxWidth: 678,
```
Comment (valevassallo) [inline frontend/src/App.js:30]: If you start writing css in a way (literal css), then I'd recommend to stick to it.

File: frontend/src/views/flight-details.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+import airplane from "../images/airplane.png";
+import { Button } from "../components/ui";
+
+function FlightDetails() {
+  return (
+    <div>
+      <img src={airplane} css={{ margin: "auto", display: "block" }} />
+      <ul css={{ padding: "1em", border: "1px solid black", margin: "1em 0" }}>
+        <li>Fliht number: 311</li>
+        <li>Capacity: 100</li>
```
Comment (valevassallo) [inline frontend/src/views/flight-details.js:13]: You can already bring this info from the store

**Review by valevassallo — APPROVED** [review]

**Review by christopher77 — APPROVED** [review]

File: frontend/src/views/flight-details.test.js (hunk 1 above)
Comment (yummta) [reply frontend/src/views/flight-details.test.js:10]: Yes, I will do it. Thanks! :)

File: frontend/src/App.js (hunk 2 above)
Comment (yummta) [reply frontend/src/App.js:30]: Yes, you right, but the library that I'm using is written in plain css. I'm going to look for another options.

File: frontend/src/views/flight-details.js (hunk 3 above)
Comment (yummta) [reply frontend/src/views/flight-details.js:13]: Yes.  I will do it! Thanks!

**Pull request merged**